src/buildHTML.js

```diff
@@ -246,374 +243,354 @@ export const makeNullDelimiter = function(options, classes) {
  * This is a map of group types to the function used to handle that type.
  * Simpler types come at the beginning, while complicated types come afterwards.
  */
-export const groupTypes = {};
+export const groupTypes = {
```

Comment:
FYI, ES6 has [a method definition shorthand](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions).

Reply:
Sorry, I didn't see it's down there. Why not for `supsub`?

Reply:
Ah, oversight. Done.